### DISCUSSION TITLE
[BUG] zabbix_return should also consider zabbix_agent2.conf

### DISCUSSION THREAD

**Discussion by Flowdalic** [saltstack/salt#67631] [Feature Requests]
As of now, zabbix_return.py only considers zabbix_agentd.conf, but for `zabbix_sender` the configuration from Zabbix agent 2, zabbix_agent2.conf, is also suitable.

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://join.slack.com/t/saltstackcommunity/shared_invite/zt-3av8jjyf-oBQ2M0vhXOhJpNpRkPWBvg)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!

**Comment by OrangeDog** [comment]
It should probably be configurable, using `config.get` and defaulting to the current path.

The formula uses the key `zabbix.lookup.agent.config` for this. Sharing it might be a good idea.

See also https://github.com/saltstack-formulas/zabbix-formula/issues/154

**Comment by Flowdalic** [comment]
Sorry, I already made a PR for this, but failed to link the PR correctly to this issue.
